muscle-quest-app/src/app/isa-level/chest-day/chest-day.component.html

```diff
@@ -0,0 +1,61 @@
+<style>
+    h1 {text-align: center;}
+    h2 {text-align: center;}
+    img {text-align: center;}
+    label {text-align: center;}
+</style>
+
+    <h1>ISA'S CHALLANGE</h1>
```

Comment:
Please fix these typos:

- challange -> challenge
- bicept -> bicep
- dumbell -> dumbbell
- shrugg -> shrug 

I would do it myself but I don't want to screw up the labels by accident.